docs/DeveloperGuide.md

```diff
@@ -31,10 +31,15 @@ Step 2: The `Command Parser` Class would run the `parse` method and create an `A
 
```

Comment:
Rather than the partial class diagram above, would it be clearer if you provided a sequence diagram instead? To show the execution of the AddItemCommand